lib/docs/scrapers/react.rb

```diff
@@ -3,7 +3,7 @@ class React < UrlScraper
     self.name = 'React'
     self.type = 'simple'
     self.release = '18.2.0'
-    self.base_url = 'https://reactjs.org/docs/'
+    self.base_url = 'https://react.dev/'
```

Comment:
> The scraper needs to be adapted/fixed!
> 
> ```
> > bundle exec thor docs:generate --force --package react
> Failed! (try running with --debug for more information)
> ```

I don't know much about the scraper but I wonder if the `base_url` needs to point to the root of the API docs?

```suggestion
    self.base_url = 'https://react.dev/reference/react/'
```